templates/documentation_layout.html

```diff
@@ -5,11 +5,15 @@
     <meta name="viewport" content="width=device-width" />
     <title>{{ page_title }}</title>
     <link rel="stylesheet" href="{{ unnest }}/index.css" type="text/css" />
+    <!-- The docs_config.js file is provided by HexDocs and shared
+         between multiple versions of the same package. -->
+    <script src="{{ unnest }}/docs_config.js"></script>
   </head>
   <body>
     <header class="page-header">
       <h2>
-        {{ project_name }}<!-- - {{ project_version }} -->
+        {{ project_name }}
+        <span id="project-version"> - v{{ project_version }} </span>
```

Comment:
Not sure about that, do we want to show the version now whenever the `docs_config.js` wasn't loaded?

Reply:
I think this is good. We can always improve it later :)